backend/fleet_management/models.py

```diff
@@ -125,3 +127,17 @@ def hash_form(initial_data: dict) -> int:
             "endMileage": initial_data["end_mileage"],
         }
         return hash_dict(required_fields)
+
+
+class Refuel(models.Model):
+    driver = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="drives_refueled"
+    )
+    car = models.ForeignKey(Car, null=False, on_delete=models.CASCADE)
+    date = models.DateField(auto_now_add=True, blank=False)
+    current_mileage = models.IntegerField(null=False)
+    refueled_liters = models.IntegerField(null=False)
+    price_per_liter = models.IntegerField(null=False)
+    currency = MoneyField(
```

Comment:
```suggestion
    price = MoneyField(
```
Seems more appropriate considering that we're tracking how much someone have spent during refuel.

Reply:
`cost` or `total_cost`?

Reply:
another thing - do we need to store `cost` (`currencty` currently), `price_per_liter` and `refueled_liters`, as it should be easily calculable as
```
cost (that probably could be <refuel_cost> = price_per_liter * refueled_liters
```

Reply:
Ha! so we had the same idea but going in two different directions!
You wanted to calculate total refuel cost whereas I wanted to calculate price per liter (https://github.com/CodeForPoznan/pah-fm/pull/523#discussion_r464920023)

Which one do you prefer?

Reply:
yes, we have the same idea, but I don't prefer either of direction here.

I believe that should be confirmed with @OlaPAH but probably drivers would fill total cost and litres refuelled and we will calculate cost per litre.